www/front_src/src/Resources/Graph/Performance/ExportableGraphWithTimeline/index.tsx

```diff
@@ -202,73 +197,36 @@ const ExportablePerformanceGraphWithTimeline = ({
       >
         <PerformanceGraph
           toggableLegend
+          additionalData={additionalData}
           adjustTimePeriod={adjustTimePeriod}
           customTimePeriod={customTimePeriod}
           displayEventAnnotations={displayEventAnnotations}
           endpoint={graphEndpoint}
           getPerformanceGraphRef={getPerformanceGraphRef}
           graphActions={
-            !isEditAnomalyDetectionDataDialogOpen && (
-              <MemoizedGraphActions
-                customTimePeriod={customTimePeriod}
-                getIsModalOpened={getIsModalOpened}
-                performanceGraphRef={
-                  performanceGraphRef as unknown as MutableRefObject<HTMLDivElement | null>
-                }
-                resourceName={resource?.name as string}
-                resourceParentName={resource?.parent?.name}
-                resourceType={resource?.type}
-                timeline={timeline}
-              />
-            )
+            <MemoizedGraphActions
+              customTimePeriod={customTimePeriod}
+              open={interactWithGraph}
+              performanceGraphRef={
+                performanceGraphRef as unknown as MutableRefObject<HTMLDivElement | null>
+              }
+              renderAdditionalGraphActions={
+                <AnomalyDetectionGraphActions
+                  details={details}
+                  resource={resource}
+                  sendReloadGraphPerformance={sendReloadGraphPerformance}
+                />
+              }
```

Comment:
Create a prop `renderAdditionalGraphActions` to pass down the component

Reply:
renderAdditionalGraphActions it's linked to GraphAction , it's the modal , by adding the props to exportableGraphTimeLine , I don't know what it will display ..